Userland/Libraries/LibWeb/WebAssembly/WebAssemblyObject.cpp

```diff
@@ -374,10 +374,13 @@ Optional<Wasm::Value> to_webassembly_value(JS::Value value, const Wasm::ValueTyp
         if (vm.exception())
             return {};
         auto value = bigint->big_integer().divided_by(two_64).remainder;
-        VERIFY(value.trimmed_length() <= 2);
-        BigEndian<i64> integer { 0 };
-        value.export_data({ &integer, 2 });
-        return Wasm::Value { static_cast<i64>(integer) };
+        VERIFY(value.unsigned_value().trimmed_length() <= 2);
+        BigEndian<u64> unsigned_integer { 0 };
+        value.unsigned_value().export_data({ &unsigned_integer, 2 * sizeof(u32) });
```

Comment:
fyi `value.unsigned_value().to_u64()` is a thing

Reply:
Ah nice, I'll use that then